src/test/java/org/mozilla/jss/tests/SSLClientAuth.java

```diff
@@ -459,15 +485,15 @@ static Extension makeBasicConstraintsExtension() throws Exception {
         OCTET_STRING enc = new OCTET_STRING(ASN1Util.encode(bc));
         return new Extension(bcOID, true, enc);
     }
-    
+
     static int nextRandInt(SecureRandom rand) throws Exception {
         int i;
         byte[] bytes = new byte[4];
         rand.nextBytes(bytes);
-        i =  ((int)bytes[0])<<24 | ((int)bytes[1])<<16 |
-                ((int)bytes[2])<<8 | ((int)bytes[3]);
+        i =  (bytes[0])<<24 | (bytes[1])<<16 |
+                (bytes[2])<<8 | (bytes[3]);
```

Comment:
I think the parentheses can be removed too, but actually there might be a flaw in this code. Java automatically promotes `byte` to `int` before the bitwise shift and or operations, so if one the byte values are negative, the sign bit would be extended, and it will override other values, thus reducing the entropy.

Let's do this change separately and probably add a test too.